package/features/steps/cli_steps.py

```diff
@@ -80,6 +80,15 @@ def install_project_requirements(context):
     """Run ``pip install -r requirements.txt``."""
     if context.kedro_version != "latest":
         requirements_path = str(context.root_project_dir) + "/src/requirements.txt"
+
+        with open(requirements_path, "r") as req_file:
```

Comment:
I would also add a comment here that this is because `numpy 2.0` breaks with old versions of `pandas` and this could be removed when the lowest version supported is updated